lineapy/graph_reader/artifact_collection.py

```diff
@@ -329,22 +388,26 @@ def generate_module(
 
         return prettify(module_text)
 
-    def get_session_module(self, dependencies: TaskGraphEdge = {}):
-        import importlib.util
-        import sys
-        from importlib.abc import Loader
+    def get_module(self, dependencies: TaskGraphEdge = {}):
+        """
+        Writing module text to a temp file and load module with names of
+        session_art1_art2_...
+        """
 
         module_name = f"session_{'_'.join(self.session_artifacts.keys())}"
-        with open(f"/tmp/{module_name}.py", "w") as f:
+        temp_folder = tempfile.mkdtemp()
+        temp_module_path = Path(temp_folder, f"{module_name}.py")
+        with open(temp_module_path, "w") as f:
             f.writelines(self.generate_module(dependencies=dependencies))
```

Comment:
rename generate_module to be `generate_module_text` to clarify the output.

Reply:
done